File: www/wpstore.js
    var exec = require('cordova/exec');

    var store_export = {};

    store_export.review = function (successCallback, errorCallback) {

        exec(function (res) {
            successCallback();
        }, errorCallback, "WPStore", "review", "");

    };

    store_export.isTrial = function (successCallback, errorCallback) {

        exec(function (res) {
            console.log("" + res);
            successCallback();
        }, errorCallback, "WPStore", "isTrial", "");

    };

    store_export.openStoreToBuyApp = function (successCallback, errorCallback) {

        exec(function (res) {
            var trialdata = JSON.parse(res);
            successCallback(trialdata);
        }, errorCallback, "WPStore", "openStoreToBuyApp", "");

    };

    module.exports = store_export;

File: src/WPStore.cs
using System.Runtime.Serialization;
using WPCordovaClassLib.Cordova;
using WPCordovaClassLib.Cordova.Commands;
using WPCordovaClassLib.Cordova.JSON;
using Microsoft.Phone.Shell;
using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.Phone.Controls;
using System.Windows;
using Microsoft.Phone.Tasks;
using Microsoft.Phone.Marketplace;

namespace WPCordovaClassLib.Cordova.Commands
{
    public class WPStore : BaseCommand
    {
    
        public void review(string options)
        {
            (new MarketplaceReviewTask()).Show();
            DispatchCommandResult();
        }

        public void openStoreToBuyApp(string options) {
            (new MarketplaceDetailTask()).Show();
            DispatchCommandResult();
        }

        public void isTrial(string options) {
            bool isTrial;
#if DEBUG
            isTrial = true; // change this enable debugging in non-trial mode
#else
            var licenseInfo = new LicenseInformation();
            isTrial = licenseInfo.IsTrial();           
#endif
            DispatchCommandResult(new PluginResult(PluginResult.Status.OK, "{\"trial\": " + isTrial.ToString().ToLower() + "}"));
        }
   
    }
}